packages/expo-dev-menu/android/src/main/java/com/facebook/react/devsupport/DevMenuSettingsBase.kt

```diff
@@ -54,6 +54,8 @@ abstract class DevMenuSettingsBase(
       mPreferences.edit().putBoolean("remote_js_debug", remoteJSDebugEnabled).apply()
     }
 
+  // TODO(kudo,20250217) - Remove this when we drop react-native 0.78 support (this came up in 0.81.0)
```

Comment:
😂

well i think we don't maintain backward compatibility much nowadays. we can just remove this method to keep code cleaner